core/src/main/java/io/questdb/PropServerConfiguration.java

```diff
@@ -703,6 +704,12 @@ public PropServerConfiguration(
             this.lineUdpUnicast = getBoolean(properties, env, "line.udp.unicast", false);
             this.lineUdpCommitMode = getCommitMode(properties, env, "line.udp.commit.mode");
             this.lineUdpTimestampAdapter = getLineTimestampAdaptor(properties, env, "line.udp.timestamp");
+            String defaultUdpPartitionByProperty = getString(properties, env, "line.udp.default.partition.by", "DAY");
+            this.lineUdpDefaultPartitionBy = PartitionBy.fromString(defaultUdpPartitionByProperty);
+            if (this.lineUdpDefaultPartitionBy == -1) {
+                log.info().$("invalid partition by ").$(lineUdpDefaultPartitionBy).$("), will use DAY for UDP").$();
+                this.lineUdpDefaultPartitionBy = PartitionBy.DAY;
```

Comment:
Can we use TCP partition strategy by default in UDP? Or better make them always same

Reply:
Using TCP settings if UDP is not specified might be confusing.
We could have just one property and use it for TCP and UDP as well, that would work better probably.
Did not go that way because we already have the tcp config which we will have to keep even if it is not documented.

Let me try to introduce a new config `line.default.partition.by` which will be used for both TCP and UDP.
If both `line.default.partition.by` and `line.tcp.default.partition.by` are present we will use line.default.partition.by.
`line.tcp.default.partition.by` will become and undocumented property.

Another option would be to keep `line.tcp.default.partition.by` and `line.udp.default.partition.by` as overrides for 'line.default.partition.by' but I would prefer just to have `line.default.partition.by`.